src/main/java/com/github/invictum/mei/Backend.java

```diff
@@ -0,0 +1,49 @@
+package com.github.invictum.mei;
+
+import com.github.invictum.mei.dtos.Queue;
+
+import java.util.List;
+import java.util.Map;
+
+public abstract class Backend {
```

Comment:
Actually backend should provide only methods for data access (e. g. getQueues and removeQueue).
Init and close is a data source abstraction responsibility.